src/plugins/workspace/public/components/workspace_creator/workspace_creator_form.tsx

```diff
@@ -176,8 +176,9 @@ export const WorkspaceCreatorForm = (props: WorkspaceCreatorFormProps) => {
                 errors={formErrors.selectedDataSources}
                 onChange={setSelectedDataSources}
                 savedObjects={savedObjects}
-                selectedDataSources={formData.selectedDataSources}
+                assignedDataSources={formData.selectedDataSources}
                 data-test-subj={`workspaceForm-dataSourcePanel`}
+                isDashboardAdmin={true}
```

Comment:
```suggestion
                isDashboardAdmin={true}
```
Is it by intension?

Reply:
Yes, it's always true here, otherwise the component wouldn't be mounted. Above, there's `isDashboardAdmin && isDataSourceEnabled && (`

Reply:
I suggest we rename `isDashboardAdmin` to `showDataSourceManagement` or something like that, because it's a UI behavior, `isDashboardAdmin` has been widely used as a term that related to permission

Reply:
But in this SelectDataSourcePanel component, `isDashboardAdmin` will continue to be passed to components like DataSourceConnectionTable, where there are some behaviors like 'select' and 'delete' do indeed need permissions verification. Using `showDataSourceManagement` during the pass may cause confusion.
In SelectDataSourcePanel:
```
  const renderTableContent = () => {
    return (
      <EuiPanel paddingSize="none" hasBorder={false}>
        <DataSourceConnectionTable
          isDashboardAdmin={isDashboardAdmin}
          dataSourceConnections={assignedDataSourceConnections}
          handleUnassignDataSources={handleUnassignDataSources}
          onSelectItems={getSelectedItems}
          inCreatePage={true}
          connectionType={AssociationDataSourceModalMode.OpenSearchConnections}
        />
      </EuiPanel>
    );
  };
```

Reply:
I see, so that's an existing props. I think you can change the `props.isDashboardAdmin` of `SelectDataSourcePanel` because this props is newly aded, let's fix this first if fixing `DataSourceConnectionTable` is out of the scope of this PR.

Reply:
ok, so in SelectDataSourcePanel it would become:
```
        <DataSourceConnectionTable
          isDashboardAdmin={showDataSourceManagement}
          ...
        />
```

Reply:
@wanglam @Kapian1234 maybe we can have a follow up task to align this prop name among these components.

Reply:
I agree that we should have a follow-up task update these components. The `isDashboardAdmin` in `DataSourceConnectionTable` controls two behaviors. The first one is to control whether table row can be selectable. The second one is to control whether data source connection can be unassociated. Shall we need break it into two properties?